README.md

```diff
@@ -189,7 +189,7 @@ python -m pip install --upgrade -r ./05_sparkplugb/requirements.txt    -e ./05_s
 # install virtual env
 python -m pip install --upgrade --user virtualenv
 python -m venv env_uns
-env_uns\Scripts\Activate.ps1
+env_uns\Scripts\Activate.ps1 ( If you get windows security error for running scripts, please run this first " powershell Set-ExecutionPolicy RemoteSigned" )
```

Comment:
Add comment with # on a new line so that code is easily copied

Reply:
Updated